### ISSUE TITLE
Window-list specific to active workspace

### ISSUE THREAD

**Issue by Techwizz-somboo** [rozniak/xfce-winxp-tc#269]
It'd be great to have an option to only show apps that are open on the active workspace on the taskbar.

**Comment by rozniak** [comment]
Is this for multiple desktops ? Might be something to consider actually, given that there was a multiple desktop powertoy for XP.. :thinking:

**Comment by Techwizz-somboo** [comment]
> Is this for multiple desktops ? Might be something to consider actually, given that there was a multiple desktop powertoy for XP.. 🤔

Yes, for multiple desktops. Even if it wasn't officially directly in XP, workspaces/desktops are very helpful and you can set keybindings to swap between them to keep your native look. That's the way I do it, but would be nice if apps from other workspaces weren't shown.

**Comment by rozniak** [comment]
Stuff that wasn't part of XP I would consider depending on what it is, though under the `wishlist` tag. I do think about it from time to time about how to handle like, 'extra' functionality (eg. tabbed file explorer).

No need to worry too much about that here though, considering there was that desktops PowerToy by MS for Windows XP that could be used as a reference for supporting the workspace/desktop bits. :grin:

**Comment by Techwizz-somboo** [comment]
> Stuff that wasn't part of XP I would consider depending on what it is, though under the `wishlist` tag. I do think about it from time to time about how to handle like, 'extra' functionality (eg. tabbed file explorer).
> 
> No need to worry too much about that here though, considering there was that desktops PowerToy by MS for Windows XP that could be used as a reference for supporting the workspace/desktop bits. 😁

Sweet, sounds great! Thanks again for your hard work!
